components/SectionFullWidthMedia.vue

```diff
@@ -21,7 +21,11 @@
     class="relative bg-primary-lighter overflow-hidden"
   >
     <!-- Background image -->
-    <VisualMedia v-if="image" :source="image" :is-background="true" />
+    <VisualMedia
+      v-if="(image && image.file) || (image && image.url)"
```

Comment:
@by-tim curious why the original check wasn't working? was the image object defined but had no file/url?

I sort of remember an issue where `v-if` returns true even if the value is `null`, was that it?

Otherwise, I think the formula is cleaner like this: `image && (image.file || image.url)`